front/src/applications/editor/nav.tsx

```diff
@@ -9,7 +9,7 @@ import { NavigateFunction } from 'react-router-dom';
 import { MapRef } from 'react-map-gl/maplibre';
 
 import { Viewport } from 'reducers/map';
-import { selectLayers } from 'reducers/editor';
+import { editorSliceActions } from 'reducers/editor';
```

Comment:
Done

Reply:
We can also "import type" : EditorContextType and Tool, SelectionState and in the lib section : MapRef, NavigateFunction, IconType and Dispatch (as suggested by @Math-R, if we could create a script that does that automatically, it would be really great))